include/deal.II/base/function.h

```diff
@@ -57,13 +57,13 @@ class TensorFunction;
  * @code
  * // access to one component at one point
  * double
- * value(const Point<dim> & p,
+ * value(const Point<dim>  &p,
```

Comment:
Because that's how I think clang-format would have formatted it, if it was regular code. But that's a slip -- it shouldn't have been included in this patch -- I'll take that out.